da/grpc/grpc.go

```diff
@@ -153,3 +153,8 @@ func (d *DataAvailabilityLayerClient) RetrieveBatches(daMetaData *da.DASubmitMet
 		Batches: batches,
 	}
 }
+
+func (d *DataAvailabilityLayerClient) GetSignerBalance() (*da.Balance, error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
TODO reminder

Reply:
i think we need to implement it otherwise it may cause e2e tests crashing when this is called.

Reply:
just return 0